fix(defaultExpandedKeys): 子节点展开后，父节点不能缩回

lianghx-319 opened the pull request:
每次折叠或者展开时更新 el-tree 的 defaultExpandedKeys
会导致 el-tree 多次去执行初始化才会执行的展开操作
实际使用的时候，并不需要外部干预 el-tree 节点展开状态

[why & how & test](https://github.com/FEMessage/el-data-tree/issues/33#issuecomment-506625937)

Closes #33

lianghx-319 commented:
**before**
- 只要子节点展开，父节点就收不起来

![201903135](https://user-images.githubusercontent.com/27187946/60327096-99388e80-99bd-11e9-855c-60d084081ed0.gif)

**after**

![201903136](https://user-images.githubusercontent.com/27187946/60327182-cf760e00-99bd-11e9-931d-6dfa58783997.gif)

donaldshen commented on src/el-data-tree.vue:
```diff
@@ -368,9 +366,7 @@ export default {
-      row: {},
-
-      expandedKeys: []
+      row: {}
```
prettierrc的trailingComma应该为"all"，受不了😖

lianghx-319 committed:
fix(#33): 二次封装干预了el-tree内部的状态管理

每次折叠或者展开时更新 el-tree 的 defaultExpandedKeys
会导致 el-tree 多次去执行初始化才会执行的展开操作
实际使用的时候，并不需要外部干预 el-tree 节点展开状态

Closes #33
